client/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+	<title>Chat prototype</title>
+	<script src="js/jquery-3.2.1.min.js"></script>
+	<script src="js/dataManager.js"></script>
+    <script src="https://unpkg.com/react@15/dist/react.js"></script>
+    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+    <script src="js/react/browser.min.js"></script>
+</head>
+<body>
+	<h1>
+		SjChat
+	</h1>
+	<script>
```

Comment:
It's better to add all scripts in separate files and include them with `<script src="js/file.js">`. The HTML and JS code becomes more readable this way.